openapi3/load_cicular_ref_with_external_file_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build go1.16
+// +build go1.16
+
+package openapi3_test
+
+import (
+	"embed"
+	"fmt"
+	"net/url"
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+//go:embed testdata/circularRef/*
+var circularResSpecs embed.FS
+
+func TestLoadCircularRefFromFile(t *testing.T) {
+	loader := openapi3.NewLoader()
+	loader.IsExternalRefsAllowed = true
+	loader.ReadFromURIFunc = func(loader *openapi3.Loader, uri *url.URL) ([]byte, error) {
+		return circularResSpecs.ReadFile(uri.Path)
+	}
+
+	doc, err := loader.LoadFromFile("testdata/circularRef/base.yml")
+	if err != nil {
+		t.Error(err)
+	}
+	fmt.Printf("%v\n", doc)
```

Comment:
Please use package `require` to assert for absence of error above and presence of correctly parsed data here. 

I'll take care of the go119 fmt changes.